src/components/incubated-by/incubated-by.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import { Logo } from "../logo/logo";
+
+import "./incubated-by.scss";
+
+export const IncubatedBy = () => {
+  return (
+    <div className="incubated-by">
+      <p className="caption">Un proiect incubat de</p>
```

Comment:
As I can see in the screenshot from issue #7  the text should be "Proiect incubat in programul" instead of "Un proiect incubat de"

Reply:
should be fine now